src/test/ui/issues/issue-18919.stderr

```diff
@@ -3,10 +3,12 @@ error[E0277]: the size for values of type `dyn for<'r> std::ops::Fn(&'r isize) -
    |
 LL | fn ho_func(f: Option<FuncType>) {
    |               ^^^^^^^^^^^^^^^^ doesn't have a size known at compile-time
+...
+LL | enum Option<T> {
+   |      ------ - required by this bound in `Option`
```

Comment:
Is HIR lowering using item `Span` instead of type param `Span` for the `T: Sized` bound?
Or I guess typeck, I forget where the responsible code is.
Maybe this can be easily improved.

Reply:
Nevermind, I missed the small space.

Reply:
Given that it confused you for a bit, should I remove the span to the ident? I want to keep it in place for degenerate cases like the following, but it might not be worth it:

```rust
enum
Option
<
T
> {}
```